pkg/apic/client.go

```diff
@@ -698,29 +698,29 @@ func (c *ServiceClient) CreateSubResourceScoped(
 	wg := &sync.WaitGroup{}
 
 	for subName, sub := range subs {
-		if strings.HasPrefix(subName, "x-") {
-			wg.Add(1)
+		// if strings.HasPrefix(subName, "x-") {
```

Comment:
@tsjohns9 @vivekschauhan @jcollins-axway  this is if statement i have to remove in order to get my subresource status acted upon.  Let me know. your thoughts on if this needs to be here since i'm piggy backing off of this.

Reply:
I think this should be removed, I think @tsjohns9 was having the same issue

Reply:
yeah I removed it for my branch